packages/frontend-2/assets/css/tailwind-setup/semantic-colors.css

```diff
@@ -1,43 +1,42 @@
+/* stylelint-disable custom-property-empty-line-before */
+/* stylelint-disable comment-empty-line-before */
 /*
   Currently not possible to define different color values for the same color name just through
   the tailwind config, so using CSS variables instead.
-
+  
+  Palette reference: https://tailwindcss.com/docs/customizing-colors
+  
   Only use colors from the default tailwind color palette so that migrating away from this (if ever needed)
-  is easy
+  is easy.
  */
 
 :root {
-  /* background */
-  --theme-color-background: #fff; /* white */
-  --theme-color-background-2: #f3f4f6; /* gray-100 */
-  --theme-color-background-3: #e5e7eb; /* gray-200 */
+  /* blank background */
+  --bg-base: #f1f5f9;
+  /* background for elements that need to sand out from the blank backgorund, e.g. cards. */
+  --bg-default: #fff;
+  /* background used to higlight elements. */
+  --bg-accent: #dbeafe;
 
-  /* foreground */
-  --theme-color-foreground: #111827; /* gray-900 */
-  --theme-color-foreground-2: #374151; /* gray-700 */
-  --theme-color-foreground-3: #6b7280; /* gray-500 */
-  --theme-color-foreground-4: #d1d5db; /* gray-300 */
+  /* default foreground color */
+  --foreground: #334155;
```

Comment:
it triggers me that foreground colors start with "foreground", but background colors start with "bg" 😄